ansible/aix61-standalone/manualBootstrap.md

```diff
@@ -243,6 +316,66 @@ pages earlier and not wait for the sync demon:
 ioo -o j2_maxRandomWrite=32
 ```
 
+## gcc 6.3.x on AIX 6.1
+
+```bash
+su - iojs
+cd /home/iojs
+
+LIBPATH=/usr/lib curl -L --insecure -O https://ci.nodejs.org/downloads/aix/V2-gcc-6.3.0-1.tar.gz
+gunzip -d V2-gcc-6.3.0-1.tar.gz
+tar -xf V2-gcc-6.3.0-1.tar
+
+LIBPATH=/usr/lib curl -L --insecure -O https://ci.nodejs.org/downloads/aix/gmake-dep.tar.gz
+gunzip -d gmake-dep.tar.gz
+tar -xf gmake-dep.tar
+```
+
+
+## Install python3
+
+AIX72: uneeded, installed by yum
```

Comment:
This line is unnecessary since it's still in the AIX 6.1 section? (Other instances seem to have been removed.)